backend/app_1/serializers.py

```diff
@@ -28,8 +28,17 @@ class Meta:
         fields = '__all__'
 
 class EatenRecordSerializer(serializers.Serializer):
+    meals = [
+        ('B', 'breakfast'),
+        ('L', 'lunch'),
+        ('D', 'dinner'),
+        ('Sn', 'snack'),
+        ('Su', 'supper')
+    ]
+
     mass = serializers.FloatField()
     date = serializers.DateTimeField(required=False, format="%Y-%m-%dT%H:%M:%SZ")
+    meal = serializers.ChoiceField(choices=meals)
```

Comment:
I think this:
```suggestion
    meal = serializers.ChoiceField(choices=ContainerMass.MealChoices.choices)
```
should work. Then you can remove the `meals` attribute above